src/plugins/auto/plugin.cpp

```diff
@@ -64,14 +64,14 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::ParseMetaDevices(cons
     // parsing the string and splitting to tokens
     std::vector<std::string> devicesWithRequests = _pluginConfig.ParsePrioritiesDevices(priorities);
 
-    auto setTputAsDefault = [&](const std::string& targetDevice,
+    auto setDefaultHint = [&](const std::string& targetDevice,
                                 std::map<std::string, std::string>& deviceConfig,
                                 const std::map<std::string, std::string>& mergedConfig) {
         auto isSetPerHint = mergedConfig.find(PluginConfigParams::KEY_PERFORMANCE_HINT) != mergedConfig.end();
         if (GetName() == "AUTO" && !isSetPerHint && mergedConfig.find(targetDevice) == mergedConfig.end()) {
-            // setting tput as the default performance mode if no hints setting for AUTO plugin and no properties
+            // setting latency as the default performance mode if no hints setting for AUTO plugin and no properties
```

Comment:
Please check if user set secondary properties like nstreams to HW device. 

If user set those secondary properties, please do not set default hint to HW device.